subliminal/core.py

```diff
@@ -523,9 +523,15 @@ def scan_videos(path, age=None, archives=True):
                 continue
 
             # skip old files
-            if age and datetime.utcnow() - datetime.utcfromtimestamp(os.path.getmtime(filepath)) > age:
-                logger.debug('Skipping old file %r in %r', filename, dirpath)
+            try:
+                file_age = datetime.utcfromtimestamp(os.path.getmtime(filepath))
+            except ValueError:
+                logger.warning('Could not get age of file %r in %r', filename, dirpath)
                 continue
+            else:
+                if file_age and age and datetime.utcnow() - file_age > age:
```

Comment:
No need to check for file_age here.

Reply:
@Diaoul I added because pycharm complained `might be referenced before assignment`